lib/pqos.h

```diff
@@ -1227,6 +1227,26 @@ int pqos_mba_ctrl_enabled(const struct pqos_cap *cap,
                           int *ctrl_supported,
                           int *ctrl_enabled);
 
+/**
+ * =======================================
+ * Vendor values
+ * =======================================
+ */
+enum pqos_vendor {
+	PQOS_VENDOR_UNKNOWN = 0, /**< UNKNOWN */
+	PQOS_VENDOR_INTEL = 1,   /**< INTEL */
+	PQOS_VENDOR_AMD = 2      /**< AMD */
```

Comment:
Please change indent to spaces. Code indent with tabs is no longer accepted in pqos.h.

Reply:
Thanks. Changed the indent to spaces now.